_tests/browser_tests/README.md

```diff
@@ -0,0 +1,92 @@
+## Quick start:
+
+_Note: If you haven't, make sure to run `./setup.sh from the project root` to install Protractor globally._
+
+From the project root:
+
+```sh
+# Update Selenium Server binaries:
+webdriver-manager update
+```
+
+From this directory:
+
+```sh
+# Start Selenium Server:
+webdriver-manager start
+
+# Open a new tab and tell protractor to start testing:
+protractor conf.js
+```
+
+If you want to test a server other than your local instance (assuming port 7000 is your local CFGov server) use the `--baseUrl` flag. Ex:
+
+```sh
+protractor conf.js --baseUrl=http://beta.consumerfinance.gov
```

Comment:
If we use an `environments.example.js` file, we risk people being out of date with the real one. If we don't commit it, Travis/Jenkins have to write the example as to `environments.js`. I don't think either of these scenarios is a good one.

I really think assuming the default local url we already call out in the [Project Readme](https://github.com/cfpb/cfgov-refresh#3-launch-sheer-to-serve-the-site) with an example of how to pass an alternate through the params is the right choice.